tests/scripts/stray_baselines.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
```

Comment:
This is something we have been using in Spicy. It mostly catches cases where a test is renamed, but the baseline is not, or the old baseline is not deleted.

It might make sense to add this as a BTest feature (e.g., emit a warning after tests are enumerated), but then again test running (via `btest`) and baselines (via `btest-diff`) are pretty decoupled.